Sources/Rendering/OpenGL/CellArrayBufferObject/index.js

```diff
@@ -261,16 +349,37 @@ function vtkOpenGLCellArrayBufferObject(publicAPI, model) {
     return cellCount;
   };
 
-  publicAPI.setCoordShiftAndScaleMethod = (shiftScaleMethod) => {
-    vtkErrorMacro('coordinate shift and scale not yet implemented');
-  };
+  publicAPI.setCoordShiftAndScale = (shifts, scales) => {
```

Comment:
You should probably have the same name between `model.coordShift` and local `shifts`. Pick the one you prefer

Reply:
Picked model.coordShift